docky/cmd/init.py

```diff
@@ -3,21 +3,28 @@
 # @author Raphaël Reverdy <raphael.reverdy@akretion.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from plumbum import cli
-from plumbum.cli.terminal import ask
-
+from plumbum import cli, local
+from plumbum.cli.terminal import prompt
 
 from .base import Docky
-from ..common.generator import GenerateProject
+from plumbum.cmd import id
 
 
 @Docky.subcommand("init")
 class DockyInit(cli.Application):
-    """Initalize a project"""
-    TEMPLATE_URL = "https://github.com/akretion/docky-odoo-template.git"
-    TEMPLATE_BRANCH = "14.0"
-    template_branch = cli.SwitchAttr("--b", help="Template's branch", argtype=str, default=TEMPLATE_BRANCH)
-    template_url = cli.SwitchAttr("--url", help="Template's url", argtype=str, default=TEMPLATE_URL)
-    
-    def main(self, *args, **kwargs):
-        GenerateProject().generate(url=self.template_url, branch=self.template_branch)
+    """Initalize the file '.env' of an existing project"""
+    def _uid(self):
+        return id['-u']().replace('\n', '')
+
+    def main(self):
+        if not local.path(".env").is_file():
+            env = prompt("Choose the env to init (dev, prod...)?", default="dev")
```

Comment:
this should be done in copier template instead

Reply:
About the .env file, I'm not sure if copier is the way to do it (maybe init it but without the real values)

IMO copier shouldn't care about the environment and is meant to be run in a dev env to initalize/update the project (and push it in git to deploy it in the other environments)

the .env file is ignored by git and should then use another path. Maybe docky is the way to do it ? (but with an adequate naming)

It feels weird to have to run copier update both on dev and prod.